src/main/java/automation/app/api/KanboardAPI.java

```diff
@@ -0,0 +1,232 @@
+        
+        return requestParams;
+    }
+
+    private static JSONObject getRequestParamsForCreateUser(String username, String password, Integer requestID) {
+        JSONObject requestParams = baseRequestParams("createUser", requestID);
+
+        Map<String, String> params = new HashMap<>();
+        params.put("username", username);
+        params.put("password", password);
+        requestParams.put("params", params);
+
+        return requestParams;
+    }
+
+    public static Response createUser(String username, String password, Integer requestID, Boolean useAuth) {
+        JSONObject requestParams = getRequestParamsForCreateUser(username, password, requestID);
+
+        return baseGetRequest(requestParams, false, useAuth);
+    }
+
+    public static Response createUserWithNoPassword(String username, Integer requestID) {
+        JSONObject requestParams = baseRequestParams("createUser", requestID);
+        
+        Map<String, String> params = new HashMap<>();
+        params.put("username", username);
+        requestParams.put("params", params);
+
+        return baseGetRequest(requestParams, false, true);
+    }
+
+    public static List<HashMap> getAllUsers() {
+        JSONObject requestParams = baseRequestParams("getAllUsers", 1000);
+
+        return baseGetRequest(requestParams, false, true).jsonPath().getList("result");
+    }
+
+    private static JSONObject getRequestParamsForRemoveUser(Integer userID, Integer requestID) {
+        JSONObject requestParams = baseRequestParams("removeUser", requestID);
+        
+        Map<String, String> params = new HashMap<>();
+        params.put("user_id", userID.toString());
+        requestParams.put("params", params);
+
+        return requestParams;
+    }
+
+    public static Response removeUser(Integer userID, Integer requestID, Boolean useAuth) {
+        JSONObject requestParams = getRequestParamsForRemoveUser(userID, requestID);
+
+        return baseGetRequest(requestParams, false, useAuth);
+
+    }
+
+    private static JSONObject getRequestParamsForCreateProject(String projectName, Integer requestID, Integer ownerID) {
+        JSONObject requestParams = baseRequestParams("createProject", requestID);
+
+        Map<String, String> params = new HashMap<>();
+        params.put("name", projectName);
+        params.put("owner_id", ownerID.toString());
+        requestParams.put("params", params);
+
+        return requestParams;
+    }
+
+    public static Response createProject(String projectName, Integer requestID, Integer ownerID, Boolean useAuth){
+        JSONObject requestParams = getRequestParamsForCreateProject(projectName, requestID, ownerID);
+
+        return baseGetRequest(requestParams, false, useAuth);
+    }
+
+    public static Response createProjectWithWrongRequestBody(Integer requestID) {
+        JSONObject requestParams = baseRequestParams("createProject", requestID);
+
+        return baseGetRequest(requestParams, false, true);
+    }
```

Comment:
This class is not the best place for method `createProjectWithWrongRequestBody` because all invalid/wrong data requests should be defined in test not on application level. Application level always must be functionally correct.
Test must be clear and readable, if you move wrong requests to methods then it assums that you need to go to method to understand what is actually sent to server